src/utils/get-signed-alice-bob-url.ts

```diff
@@ -1,16 +1,33 @@
 import axios from 'axios';
 import crypto from 'crypto';
 
-interface AliceBobExchangeInfo {
+export interface AliceBobExchangeInfo {
   from: string;
   to: string;
   fromAmount: number;
-  userId: string;
-  toPaymentDetails: string;
+  userId?: string;
+  toPaymentDetails?: string;
 }
 
 export const getSignedAliceBobUrl = async (exchangeInfo: AliceBobExchangeInfo) => {
   const now = +new Date();
+  const signature = getAliceBobSignature(exchangeInfo, now);
+
+  const response = await axios.post<{ payUrl: string }>(
+    'https://api.abex.pro/api/v3/create-order',
+    exchangeInfo,
+    {
+      headers: {
+        'public-key': process.env.ALICE_BOB_PUBLIC_KEY!,
+        'timestamp': now,
+        signature
+      }
+    });
+
+  return response.data.payUrl;
+};
+
+export const getAliceBobSignature = (exchangeInfo: AliceBobExchangeInfo, now: number) => {
```

Comment:
`now` param can be something like `traceTimestamp` and be created at the beginning of first handler at controller. It's very similar to traceId right now, because created once and reused in other methods. Mb good for // TODO: add trace-logs